doc/man3/EVP_des_cbc.pod

```diff
@@ -54,6 +54,10 @@ EVP_des_ofb()
 DES in CBC, ECB, CFB with 64-bit shift, CFB with 1-bit shift, CFB with 8-bit
 shift and OFB modes.
 
+None of these algorithms are provided by OpenSSL default provider.
+To use them it is necessary to load either legacy OpenSSL provider or another
```

Comment:
_the OpenSSL legacy provider_ instead of _legacy OpenSLL provider_.

Reply:
Done. Thanks!